v2/spanner-to-sourcedb/README.md

```diff
@@ -0,0 +1,363 @@
+
+  For example:
+
+  ```
+  If the GCS DLQ bucket is : gs://rr-dlq
+  PubSub Topic is : projects/<my-project>/topics/rr-dlq-topic
+  Pubsub Subscription is : projects/<my-porject>/subscriptions/rr-dlq-subs
+
+  Then configure the notification like:
+
+
+  gcloud storage buckets notifications create --event-types=OBJECT_FINALIZE
+  --object-prefix=retry/ gs://rr-dlq --topic=projects/<my-project>/topics/rr-dlq-topic
+  ```
+15. Create change stream, the below example tracks all tables. When creating a change stream, use the NEW_ROW option, sample command below :
+
+  ```
+  CREATE CHANGE STREAM allstream
+  FOR ALL OPTIONS (
+  retention_period = '7d',
+  value_capture_type = 'NEW_ROW'
+  );
+  ```
+16. The Dataflow template creates a pool of database connections per Dataflow worker. The maxShardConnections template parameter, defaulting to 10,000 represents the maximum connections allowed for a given database. The maxWorkers Dataflow configuration should not exceed the maxShardConnections value, else the template launch will fail as we do not want to overload the database.
+
+17. Please refer dataflow [documentation](https://cloud.google.com/dataflow/docs/guides/routes-firewall#internet_access_for) on network options.
+
+  When disabling the public IP for Dataflow, the option below should be added to the command line:
+
+  ```
+  --disable-public-ips
+  ```
+
+  When providing subnetwork, give the option like so:
+
+  ```
+  --subnetwork=https://www.googleapis.com/compute/v1/projects/<project name>/regions/<region name>/subnetworks/<subnetwork name>
+  ```
+
+### Sample source shards File
+
+This file contains meta data regarding the source MYSQL shards, which is used to connect to them. This should be present even if there is a single source database shard.
+The database user password should be kept in [Secret Manager](#https://cloud.google.com/security/products/secret-manager) and it's URI needs to be specified in the file.
+The file should be a list of JSONs as:
+
+```json
+[
+    {
+    "logicalShardId": "shard1",
+    "host": "10.11.12.13",
+    "user": "root",
+    "secretManagerUri":"projects/123/secrets/rev-cmek-cred-shard1/versions/latest",
+    "port": "3306",
+    "dbName": "db1"
+    },
+    {
+    "logicalShardId": "shard2",
+    "host": "10.11.12.14",
+    "user": "root",
+    "secretManagerUri":"projects/123/secrets/rev-cmek-cred-shard2/versions/latest",
+    "port": "3306",
+    "dbName": "db2"
+    }
+]
+```
+
+## Launching reverse replication
+
+Currently, the reverse replication flow is launched manually. Please refer the Dataflow template [readme](https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/spanner-to-sourcedb/README_Spanner_to_SourceDb.md).
```

Comment:
Doesn't the link refer to the same file? I didn't find any sample command to launch the job

Reply:
It refers to the generated readme file - which has commands to build and run

Reply:
okay, i got confused, thanks for clarifying